orders/src/main/java/com/orders/crudyorders/model/Customer.java

```diff
@@ -0,0 +1,137 @@
+package com.orders.crudyorders.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "customer")
+public class Customer
```

Comment:
Line 6 - It is advisable to make table names plural in order to reduce the likelihood of naming conflicts (the class name below can still be singular).